packages/hs-react-ui/src/components/Table/Table.stories.tsx

```diff
@@ -35,7 +50,18 @@ const NoteField = styled.textarea`
   resize: none;
 `;
 
-const generateSampleData = (rows: number) => {
+const groupLabelPositionOptions: { above: 'above'; below: 'below' } = {
+  above: 'above',
+  below: 'below',
+};
```

Comment:
Does this need to be typed?

Reply:
They need to in order to pass the linting on the build. The inputs for the label position are typed as one of the strings `above` or `below` and without the typing they get flagged as type string rather than `above` or `below`